src/main/java/com/maxmind/minfraud/exception/HttpException.java

```diff
@@ -11,7 +11,13 @@
  * MinFraudException.
  */
 public final class HttpException extends IOException {
+    /**
+     * httpStatus 
```

Comment:
I am not sure documenting private fields like this makes sense, especially when the documentation isn't adding anything.